blueprints/deprecation/index.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-env node */
+module.exports = {
+  description: 'Generate a new deprecation file',
+
+  locals(options) {
+    // Return custom template variables here.
+    return {
+      name: extractDeprecationId(options.entity.name)
+    };
+  }
+};
+
+function extractDeprecationId(path) {
+  var pathParts = path.split('/')
+  var name = pathParts[pathParts.length - 1].split('.')[0]
```

Comment:
it's not quite obvious what this code extracts. since there are no tests for this it should at least have some comment with an example that explains the behavior.

Reply:
Can you write the test? 😅